docs/about.md

```diff
@@ -55,13 +55,50 @@ Contributor [2021 January - ]<br/>
 
 <!-- ------------------------------------------------------------------------------------------------------ -->
 
-### [Gokul Rajiv](https://github.com/gok99)
+### [Zhou Jiahao](https://github.com/zhoukerrr)
 **Role**: Contributor [2022 January - ]<br/>
 
 <!-- ------------------------------------------------------------------------------------------------------ -->
 
-### [Zhou Jiahao](https://github.com/zhoukerrr)
-**Role**: Contributor [2022 January - ]<br/>
+### [Charisma Kausar](https://www.github.com/ckcherry23)
+**Role**: Committer [2024 January - ]<br/>
+Contributor [2022 December - ]<br/>
+
+<!-- ------------------------------------------------------------------------------------------------------ -->
+
+### [David Gareth Ong](https://www.github.com/vvidday)
+**Role**: Committer [2024 January - ]<br/>
```

Comment:
All mentors were contributors too! (So update for David and Marcus)
```suggestion
**Role**: Committer [2024 January - ]<br/>
Contributor [2023 January - ]<br/>
```

Reply:
I see, will do!